.github/workflows/build.yml

```diff
@@ -465,6 +467,7 @@ jobs:
           chmod uog+x build/moonbeam
           cd moonbeam-types-bundle
           npm install
+          npm run build
```

Comment:
This is useless no ? there is a `rm` right after

Reply:
you're right